packages/e2e/features/ui/components/authenticator/sign-in-sms-mfa.feature

```diff
@@ -7,16 +7,25 @@ Feature: Sign In with SMS MFA
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-sms-mfa"
 
-  @next @react @vue @skip
+  @angular @next @react @vue @skip
   Scenario: Sign in using a valid phone number and SMS MFA
     When I type a valid phone number "VALID_PHONE_NUMBER"
     And I type a valid password "VALID_PASSWORD"
     And I click the "Sign In" button
     Then I will be redirected to the confirm sms mfa page
     
-  @next @react @angular
+  @angular @next @react
   Scenario: Sign in with invalid credentials
     When I type an invalid username "INVALID_PHONE_NUMBER"
     And I type an invalid password "INVALID_PASSWORD"
     And I click the "Sign In" button
     Then I see "User does not exist"
+
+  # This feature is not yet implemented
+  @skip
+  Scenario: Expired SMS code
+    When I type a valid pone number "VALID_PHONE_NUMBER"
+    And I type a valid password "VALID_PASSWORD"
+    And I click the "Sign In" button
+    And I enter my confirmation code after some delay
+    Then I see 'Resend Code'
```

Comment:
Not possible with current APIs, AFAIK

Reply:
I thought we already had Resend Code on this screen (without the need to enter a confirmation code)

https://github.com/aws-amplify/amplify-js/blob/836c94aca1676fcd1684f9fcae9587a2da14945e/packages/amplify-ui-components/src/components/amplify-confirm-sign-up/amplify-confirm-sign-up.tsx#L227-L241

Reply:
Different thing. That's resending a confirm sign up code from Cognito, not an SMS MFA code.

Reply:
Ahh thanks! All these flows are blurring together in my mind 😰